### PR TITLE
Show config on Windows CI

### PR DESCRIPTION
**Pull request by leofang**

### CODE REVIEW

**Comment by leofang** [comment]
Jenkins, test this please

**Comment by kmaehashi** [comment]
I think it's done at line 67?

**Comment by leofang** [comment]
Yes, I missed that 😅

**Comment by leofang** [comment]
@kmaehashi Can I bring back this? I feel it's easier if I can see the `show_config` outcome at the start of the test log...

File: .pfnci/windows/test.ps1 [hunk 1]
```diff
@@ -106,6 +106,7 @@ function Main {
     }
     echo "Running test..."
     $test_retval = 0
+    python -c "import cupy; cupy.show_config()" > cupy_test_log.txt
     python -m pytest -rfEX @pytest_opts tests > cupy_test_log.txt
```
Comment (kmaehashi) [inline .pfnci/windows/test.ps1:110]:
```suggestion
    python -m pytest -rfEX @pytest_opts tests >> cupy_test_log.txt
```

Comment (leofang) [reply .pfnci/windows/test.ps1:110]: Ah...Silly me. Thanks, @kmaehashi.

**Review by kmaehashi — APPROVED** [review]
LGTM!

**Comment by leofang** [comment]
Windows CI is green, with the config correctly prepended in the test log:
https://storage.googleapis.com/chainer-artifacts-pfn-public-ci/cupy-ci/70099/cupy_test_log.txt

**Pull request merged**